docs/source/configuration_file.rst

```diff
@@ -0,0 +1,364 @@
+``aleatoric`` can be ``true``. - ``epistemic``: Bool. Model-based
+uncertainty with `Monte Carlo
+Dropout <https://arxiv.org/abs/1506.02142>`__. - ``aleatoric``: Bool.
+Image-based uncertainty with `test-time
+augmentation <https://doi.org/10.1016/j.neucom.2019.01.103>`__. -
+``n_it``: Integer. Number of Monte Carlo iterations. Set to 0 for no
+uncertainty computation.
+
+Cascaded Architecture Features
+------------------------------
+
+object\_detection\_params (Optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+-  ``object_detection_path``: String. Path to object detection model.
+   The model's prediction will be used to generate bounding boxes.
+-  ``safety_factor``: List. List of length 3 containing the factors to
+   multiply each dimension of the bounding box. Ex: If the original
+   bounding box has a size of 10x20x30 with a safety factor of [1.5,
+   1.5, 1.5], the final dimensions of the bounding box will be 15x30x45
+   with an unchanged center.
+
+Transformations
+---------------
+
+Transformations applied during data augmentation. Transformations are
+sorted in the order they are applied to the image samples. For each
+transformation, the following parameters are customizable: -
+``applied_to``: list betweem ``"im", "gt", "roi"``. If not specified,
+then the transformation is applied to all loaded samples. Otherwise,
+only applied to the specified types: eg ``["gt"]`` implies that this
+transformation is only applied to the ground-truth data. -
+``dataset_type``: list between ``"training", "validation", "testing"``.
+If not specified, then the transformation is applied to the three
+sub-datasets. Otherwise, only applied to the specified subdatasets: eg
+``["testing"]`` implies that this transformation is only applied to the
+testing sub-dataset.
+
+Available transformations:
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+-  ``NumpyToTensor``
+-  ``CenterCrop2D`` (parameters: ``size``)
+-  ``ROICrop2D`` (parameters: ``size``)
+-  ``NormalizeInstance``
+-  ``RandomAffine`` (parameters: ``degrees`` (Positive integer),
+   ``translate`` (List of floats between 0. and 1.), ``scale`` (List of
+   floats between 0. and 1.))
+-  ``RandomShiftIntensity`` (parameters: ``shift_range``)
+-  ``ElasticTransform`` (parameters: ``alpha_range``, ``sigma_range``,
+   ``p``)
+-  ``Resample`` (parameters: ``wspace``, ``hspace``, ``dspace``)
+-  ``AdditionGaussianNoise`` (parameters: ``mean``, ``std``)
+-  ``DilateGT`` (parameters: ``dilation_factor``) Float. Controls the
+   number of iterations of ground-truth dilation depending on the size
+   of each individual lesion, data augmentation of the training set. Use
+   ``0`` to disable.
+-  ``HistogramClipping`` (parameters: ``min_percentile``,
+   ``max_percentile``)
+-  ``Clage`` (parameters: ``clip_limit``, ``kernel_size``)
+-  ``RandomReverse``
+
+Examples
+--------
+
+Examples of configuration files: `config\_config.json <ivadomed/config/config.json>`__.
```

Comment:
so, in the built docs of the temporary branch, i get this missing page error: https://ivadomed.org/en/aj-286-convert-docs-to-rst/ivadomed/config/config.json

however, i am not sure if this link will work once merged on master? (i suspect if it fails on the branch, it will also fail on master)

Reply:
Indeed this is broken (just like it is on the current master branch), I had created https://github.com/neuropoly/ivadomed/issues/298 for this purpose